.github/workflows/build-nethermind-docker-images.yml

```diff
@@ -1,20 +1,35 @@
 name: '[BUILD] Docker images and publish'
 
 on:
+  push:
```

Comment:
Let's change to:

```yaml
push:
  branches: [master]
paths:
  - 'src/Nethermind/**'
```

Reply:
What about adding
```yaml
- '!src/Nethermind/*.Test'
```

Reply:
This will limit this one a bit.
but I meant that look below - I'd add some change to yaml and all nethermind unit tests will be triggered - this is worse than those micro optimizations :)

Reply:
I thought you mean you don't wanna trigger Docker build on any test commit. Well, if you mean that each PR commit triggers required tests, then it is what it is. As they are required, they must run anyway. Otherwise, we need to make them not required.